hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/MockNodeManager.java

```diff
@@ -267,6 +267,19 @@ public void removePipeline(Pipeline pipeline) {
     node2PipelineMap.removePipeline(pipeline);
   }
 
+  @Override
+  public void addContainer(DatanodeDetails dd,
+                           ContainerID containerId)
+      throws NodeNotFoundException {
+    try {
+      Set<ContainerID> set = node2ContainerMap.getContainers(dd.getUuid());
+      set.add(containerId);
+      node2ContainerMap.setContainersForDatanode(dd.getUuid(), set);
+    } catch (SCMException e) {
+      e.printStackTrace();
```

Comment:
Is this needed ? can we just throw the exception or log the error.

Reply:
We can only throw `NodeNotFoundException` or any runtime exception.
> log the error.
`printStackTrace` was added to log the error :) We don't have any logger defined in MockNodeManager.

The test-case using `MockNodeManager` will anyways depend on the existence of container after calling addContainer, and the test should eventually fail if this call fails.